src/Headers.ts

```diff
@@ -4,6 +4,8 @@ import { normalizeHeaderValue } from './utils/normalizeHeaderValue'
 
 const NORMALIZED_HEADERS: unique symbol = Symbol('normalizedHeaders')
 const RAW_HEADER_NAMES: unique symbol = Symbol('rawHeaderNames')
+const HEADER_JOIN_DELIMITER = ', ' as const
+const SET_COOKIE_HEADER_KEY_NORMALIZED = 'set-cookie' as const
```

Comment:
`SET_COOKIE_HEADER_NAME`?

Reply:
Also, we seem to be using this only in the `getSetCookie()` method. What do you think if we don't extract it to a constant just yet?